tests/App/AppKernel.php

```diff
@@ -43,9 +52,17 @@ public function registerBundles(): iterable
     {
         return [
             new FrameworkBundle(),
+            new SecurityBundle(),
             new TwigBundle(),
+            new DoctrineBundle(),
+            new KnpMenuBundle(),
             new FOSCKEditorBundle(),
+            new SonataFormBundle(),
+            new SonataTwigBundle(),
             new SonataBlockBundle(),
+            new SonataDoctrineBundle(),
+            new SonataAdminBundle(),
+            new SonataDoctrineORMAdminBundle(),
```

Comment:
I need to add a LOT of dependencies to make this work...

Because I need to run SonataMediaBundle, and AdminBundle and DoctrineORM, and it brings a lot of dependencies. I have to also add them on composer require-dev section. Do you agree @VincentLanglet ?

Reply:
Seems ok to me. I'm not surprised by such needs